weave_query/MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include requirements.*
```

Comment:
Why are we putting this here? weave_query is not a python package that gets built, right?

Reply:
We could if you are confident we will never build this pkg again.  You can always go into git history to find this file later if you need it.

Then instead of move, these will be deletes

Reply:
i see, probably safe to keep